src/index.js

```diff
@@ -219,12 +219,13 @@ document.addEventListener("DOMContentLoaded", () => {
     const paramesDict = { 'searchFields': decodedSearchFields, 'sortFields': decodedSortFields };
 
     // Use solrClient built-in functions
+    // set view facet
+    solrClient.setView(decodedView);
```

Comment:
We did store the viewpoint information in the URL parameter but the order to change the redux store caused the Cesium map only renders based on the initial camera setup.